src/equation_of_state.hpp

```diff
@@ -54,9 +57,15 @@ static const int MAXSPECIES = 200;
 static const double UNIVERSALGASCONSTANT = 8.3144598;  // J * mol^(-1) * K^(-1)
 static const double AVOGADRONUMBER = 6.0221409e+23;    // mol^(-1)
 static const double BOLTZMANNCONSTANT = UNIVERSALGASCONSTANT / AVOGADRONUMBER;
+static const double PLANCKCONSTANT = 6.62607015e-34;  // m^2 kg / s
 static const double VACUUMPERMITTIVITY = 8.8541878128e-12;
 static const double ELECTRONCHARGE = 1.60218e-19;
 static const double MOLARELECTRONCHARGE = ELECTRONCHARGE * AVOGADRONUMBER;
+static const double ELECTRONMASS = 9.1093837015e-31;  // kg
+// static const double qeOverkB = ELECTRONCHARGE / BOLTZMANNCONSTANT;
+
+static const double IonizationEnergy_Argon = 13.598434599702;  // eV
```

Comment:
Check where this is used and check the value.  I thought the argon ionization energy was closer to 15.76 eV?